notebooks/official/model_evaluation/automl_tabular_regression_model_evaluation.ipynb

```diff
@@ -0,0 +1,1485 @@
+{
```

Comment:
<p>make it a habit to remove the trailing comma on the last parameter</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/940/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Reply:
<p>I removed trailing comma on the last parameter. But when I run linter test, comma is added automatically</p>